CHANGELOG.md

```diff
@@ -31,6 +31,8 @@
     * Update PayPal app URL query scheme from `paypal-app-switch-checkout` to `paypal`
   * BraintreeAmericanExpress
     * Update initializer to `BTAmericanExpressClient(authorization:)`
+  * BraintreePayPal
```

Comment:
It looks like we have 3 `BraintreePayPal` sections under breaking changes. Lets put them all under 1 header vs 3.